fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -263,10 +264,18 @@ private void parseProperties() throws DdlException {
 
         if (properties.containsKey(PROPERTY_CSV_COLUMN_SEPARATOR)) {
             csvColumnSeparator = properties.get(PROPERTY_CSV_COLUMN_SEPARATOR);
+            int len = csvColumnSeparator.getBytes(StandardCharsets.UTF_8).length;
+            if (len > 50 || len == 0) {
+                throw new DdlException("the length of csv.column_separator property should be in [1, 50]");
```

Comment:
the error message is `xxx is limited to a maximum of 50 bytes` in `StreamLoadScanNode` and `FileScanNode` and  better use the same error message.

put the error message in `ErrorCode`?